styleguide/_layouts/default.jade

```diff
@@ -82,3 +82,6 @@ html(lang="en")
   script(src="https://577351e10bc4a0198d93-f60a0bb748a3c84145bb10da7563bafb.ssl.cf1.rackcdn.com/zoolander/bootstrap-tabcollapse.js")
   script.
     $(".imacBanner-headline").fitText(1.1, { maxFontSize: '45px' });
+  script(src="js/classie.js")
```

Comment:
we talked about this separately, but just including for the sake of the review. Since we're already somewhat coupled to jQuery, let's just rely on that for now. If we decide to remove jQuery this may be a direction to go with in the future, but I'd rather not add any additional libraries to the code if we have an alternative already built in.